src/components/Notification/Notification.js

```diff
@@ -0,0 +1,44 @@
+import ClassNames from 'classnames';
+import React from 'react';
+import PropTypes from 'prop-types';
+import Spinner from '../Spinner/Spinner';
+
+const Notification = ({ type, children, seen, className, ...props }) => {
+  const classes = ClassNames(
+    {
+      'drawer-pf-notification': type === 'notification',
+      'drawer-pf-loading text-center': type === 'loading'
+    },
+    { unread: !seen },
+    className
+  );
+
+  return (
+    <div className={classes} {...props}>
+      {type === 'loading'
+        ? [
+            <Spinner key="notification_spinner" inline loading size="xs" />,
+            ' Loading More'
+          ]
+        : children}
+    </div>
```

Comment:
In your storybook, it seems like there is a missing horizontal line under the spinner:
<img width="325" alt="screen shot 2018-03-21 at 7 22 14 pm" src="https://user-images.githubusercontent.com/12504403/37744531-ab22c13a-2d3d-11e8-8af4-6018c095dd86.png">
below is the core example:
<img width="327" alt="screen shot 2018-03-21 at 7 22 04 pm" src="https://user-images.githubusercontent.com/12504403/37744529-a6965e06-2d3d-11e8-85c0-31bb168b3524.png">